admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/utils/MasterdataSearchHelperTest.java

```diff
@@ -246,28 +263,481 @@ public void searchwildcardSearchMasterdata() {
+	}
+	
+	@Test
+	public void filterTypesTest02(){
+		SearchFilter filter = new SearchFilter();
+		filter.setValue("123");
+		filter.setType("filter");
+		ReflectionTestUtils.invokeMethod(searchHelper,"FilterTypes",filter);
+	}
+	
+	@Test
+	public void validateSort01(){
+		SearchSort searchSort = new SearchSort("id","asc");
+		ReflectionTestUtils.invokeMethod(searchHelper,"validateSort",searchSort);
+	}
+	
+	@Test (expected = RequestException.class)
+	public void validateSort02(){
+		SearchSort searchSort = new SearchSort(null,null);
+		ReflectionTestUtils.invokeMethod(searchHelper,"validateSort",searchSort);
+		assertEquals(null,searchSort);
+	}
+	
+	@Test (expected = NullPointerException.class)
+	public void nativeMachineQuerySearch01(){
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode();
+		List<Zone> zones = new ArrayList<>(2);
+		searchHelper.nativeMachineQuerySearch(searchDtoWithoutLangCode,"filter",zones,false);
+	}
+	
+	@Test (expected = NullPointerException.class)
+	public void nativeMachineQuerySearch02(){
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode();
+		List<Zone> zones = new ArrayList<>();
+		searchHelper.nativeMachineQuerySearch(searchDtoWithoutLangCode,"filter",zones,true);
+	}
+	
+	@Test (expected = NullPointerException.class)
+	public void nativeMachineQuerySearch03(){
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode();
+		List<Zone> zones = new ArrayList<>();
+		Iterator<SearchFilter> searchIterator = searchDtoWithoutLangCode.getFilters().iterator();
+		SearchFilter searchFilter = searchIterator.next();
+		SearchFilter searchFilter1 = searchFilter;
+		searchHelper.nativeMachineQuerySearch(searchDtoWithoutLangCode,"searchFilter",zones,true);
+	}
+	
+	@Test (expected = InvalidDataAccessApiUsageException.class)
+	public void nativeMachineQuerySearchIterator(){
+		SearchDtoWithoutLangCode searchDtoWithoutLangCode = new SearchDtoWithoutLangCode();
+		List<SearchFilter> filters = new ArrayList<>();
+		SearchFilter filter1 = new SearchFilter();
+		filter1.setColumnName("name");
+		filter1.setValue("value");
+		filter1.setType("equals");
+		filters.add(filter1);
+		searchDtoWithoutLangCode.setFilters(filters);
+		Pagination pagination = new Pagination();
+		pagination.setPageStart(0);
+		pagination.setPageFetch(10);
+		searchDtoWithoutLangCode.setPagination(pagination);
+		String typeName = "typeName";
+		List<Zone> zones= new ArrayList<>();
+		Zone zone1 = new Zone();
+		zone1.setCode("java");
+		zone1.setLangCode("eng");
+		zone1.setHierarchyPath("hierarchy");
+		zone1.setHierarchyLevel((short)1);
+		zone1.setParentZoneCode("Parent");
+		zone1.setName("name");
+		zone1.setHierarchyName("name");
+		zone1.setIsActive(true);
+		zones.add(zone1);
+		boolean isAssigned = true;
+		searchHelper.nativeMachineQuerySearch(searchDtoWithoutLangCode,"filter",zones,isAssigned);
+	}
+	
+	@Test (expected = NullPointerException.class)
```

Comment:
I'm still able to see a lot of testmethods expecting  nullpointer exception. Please check that.

Reply:
Removed All NullPointerExceptions